packages/manager/apps/freefax/src/index.html

```diff
@@ -4,6 +4,7 @@
         <title>FreeFax App</title>
```

Comment:
So we need translation ?

Reply:
App is not currently available to customer so it is not an issue the title is not translated yet. Also I think it is out of scope of this PR as main point is to add the breadcrumb